### PR TITLE
Add new map control `zoomhistory`

### LINKED ISSUE #387: Zoom/Pan History control
### Checklist

- [X] I've searched through the [current issues](https://github.com/g3w-suite/g3w-client/issues) to make sure this feature hasn't been requested already.

### Motivation

It could be useful to have a new map control to move between map/pan history as QGIS do

Screenshot from 2023-03-21 14-51-17

### Suggested solution

Implement a map control that store zoom/pan client action

### Alternatives considered

_No response_

### PR DESCRIPTION
**Pull request by volterra79**
Closes: #387

Steps:

1. From G3W-ADMIN click on **tools icon** (top-right) 
Screenshot from 2023-03-21 14-55-56
and click on **Django Administration** item
Screenshot from 2023-03-21 14-55-02

2. Click on **Map controls** item 

Screenshot from 2023-03-21 14-58-01

3. Click on **ADD MAP CONTROL**

Screenshot from 2023-03-21 14-59-47

4. Set _zoomhistory_ in name input and Save it

Screenshot from 2023-03-21 15-01-02

5.  Move it to the top and back to G3W-ADMIN

Screenshot from 2023-03-21 15-06-26

6. Go to Cartographic Group and chose a Group where add new map control

Screenshot from 2023-03-21 15-02-48

7. It will appear on G3W-ClIENT

Screenshot from 2023-03-21 15-09-10

### CODE REVIEW

**Comment by wlorenzetti** [comment]
I'll open a new issue to add by default `zoomhistory` to g3w-admin.

**Comment by volterra79** [comment]
> I'll open a new issue to add by default `zoomhistory` to g3w-admin.

@wlorenzetti  Please add also _querybydrawpolygon_  as default map control https://github.com/g3w-suite/g3w-client/pull/341

**Comment by wlorenzetti** [comment]
https://github.com/g3w-suite/g3w-admin/issues/486

File: src/components/MapControlZoomHistory.vue [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  <div
+    style="display:flex; flex-direction: column"
+    class="ol-zoom-history ol-unselectable ol-control">
+    
+    <!-- STEP BACK -->
+    <button
+      @click.stop.prevent="history.index--"
+      type="button"
+      v-disabled="history.index === 0"
+      v-t-tooltip="'Back'"
```
Comment (Raruto) [inline src/components/MapControlZoomHistory.vue]: **NB** other translations are still missing, 

Something like `"step back"` and `"step forward"` could be more appropriate?

**Review by Raruto — CHANGES_REQUESTED** [review]

**Review by Raruto — APPROVED** [review]

**Review by Raruto — APPROVED** [review]
Not a big fan of those arrow icons, but as a starting point it's fine

**Comment by volterra79** [comment]
> Not a big fan of those arrow icons, but as a starting point it's fine

@Raruto we can choose icons here https://fontawesome.com/v5/search?o=r&m=free

**Comment by Raruto** [comment]
Perhaps setted as follows it might be more immediate to understand, but as i often say right now i don't know how much "rework" it might require..

image

**Font awesome icons used:**

- [reply](https://fontawesome.com/v5/icons/reply?f=classic&s=solid)
- [share](https://fontawesome.com/v5/icons/share?f=classic&s=solid)

**Comment by Raruto** [comment]
@volterra79 great job with https://github.com/g3w-suite/g3w-client/pull/388/commits/6a72558e94d192ff9ee6916ad3730097b59605a7 🎖️.

Feel free to merge it.

**Pull request merged**